RecoLocalCalo/HGCalRecProducers/plugins/KernelManagerHGCalCellPositions.cu

```diff
@@ -0,0 +1,23 @@
+#include <cuda.h>
+#include <cuda_runtime.h>
+#include <inttypes.h>
+#include "RecoLocalCalo/HGCalRecProducers/plugins/KernelManagerHGCalCellPositions.h"
+#include "RecoLocalCalo/HGCalRecProducers/plugins/HGCalCellPositionsKernelImpl.cuh"
+
+KernelManagerHGCalCellPositions::KernelManagerHGCalCellPositions(const size_t& nelems) {
+  ::nb_celpos_ = (nelems + ::nt_celpos_.x - 1) / ::nt_celpos_.x;
+}
+
+void KernelManagerHGCalCellPositions::fill_positions(
+    const hgcal_conditions::HeterogeneousHEFCellPositionsConditionsESProduct* d_conds) {
+  fill_positions_from_detids<<<::nb_celpos_, ::nt_celpos_>>>(d_conds);
+  cudaCheck(cudaDeviceSynchronize());
```

Comment:
Can this be dropped?

Reply:
No, since it represents the fundamental step to fill the detids of HGCAL cells directly in the GPU (implemented in ```RecoLocalCalo/HGCalRecProducers/plugins/HGCalCellPositionsKernelImpl.cu```). The fill could be done in the CPU, and copied later to the device, but this should be even better.
Are you perhaps referring only to the synchronization step? That I believe should indeed be removed.

Reply:
I meant only the `cudaDeviceSynchronize()`, sorry for being unclear.